File: src/mffs/item/card/ItemCardInfinite.java
package mffs.card;

import mffs.api.card.ICardInfinite;

/**
 * A card used by admins or players to cheat infinite energy.
 * 
 * @author Calclavia
 * 
 */
public class ItemCardInfinite extends ItemCard implements ICardInfinite
{
	public ItemCardInfinite(int id)
	{
		super(id, "cardInfinite");
	}
}

File: src/main/scala/mffs/item/card/ItemCardInfinite.scala
package mffs.item.card

import nova.core.fluid.{Fluid, TankSimple}

/**
 * A card used by admins or players to cheat infinite energy.
 *
 * @author Calclavia
 */
class ItemCardInfinite extends ItemCard {
	val tank = new TankSimple(Fluid.bucketVolume)

	add(tank)

	override def getID: String = "cardInfinite"
}